src/Cats.js

```diff
@@ -0,0 +1,40 @@
+import './App.css';
+
+
+function Cats() {
+  return (
+
+    <div>
+
+      <div className='Cat-header'>
```

Comment:
Hey! It's best practice to use PascalCase when making class names in React! https://stackoverflow.com/questions/57221878/react-classname-naming-convention